spec/controllers/submissions_controller_spec.rb

```diff
@@ -958,6 +962,23 @@
             expect(response).to render_template(partial: 'shared/_poll_job')
           end
 
+          it 'should succeed if it is before the section due date and collect_current is selected' do
+            @assessment_section_properties.update!(due_date: 1.week.from_now)
+            allow(Assignment).to receive_message_chain(
+              :includes, :find
+            ) { @assignment }
+            expect_any_instance_of(SubmissionsController).not_to receive(:flash_now).with(:error, anything)
```

Comment:
can we split this test up into two. I'd prefer to have a single expect statement per `it` block whenever possible